pom.xml

```diff
@@ -59,6 +60,13 @@
     </dependencies>
```

Comment:
Nitpick: I think the indentation on this closing tag is a little off?

Reply:
this is weird, the indentation is fine for me locally, can you check what it looks like in your intellij for me?

Reply:
All looks normal on IntelliJ 👍